doc/changelog/02-specification-language/19833-evd-inst-beta.rst

```diff
@@ -0,0 +1,4 @@
+- **Changed:**
+  Reduce beta-redexes when failing to instantiate an evar
```

Comment:
Be more declarative so users who don't know the algorithm can still understand the change
```suggestion
  Unification will now reduce beta-redexes when necessary to instantiate evars
```